src/compiler.js

```diff
@@ -480,6 +495,64 @@ class Compiler {
               )
         })
 
+        if (subgraph.has('mutations')) {
+          // Copy mutation files and update their paths
+          subgraph = subgraph.update('mutations', mutations =>
+            // mutations/schema.graphql + mutations/resolvers/index.js
+
+            mutations
+              // Write the mutation schema + root schema to the output directory
+              .updateIn(['schema', 'file'], schemaFile => {
+                const rootSchemaFile = path.resolve(
+                  this.sourceDir, subgraph.getIn(['schema', 'file'])
+                )
+                schemaFile = path.resolve(this.sourceDir, schemaFile)
+
+                // Concatenate the schemas
+                let schemaData = fs.readFileSync(rootSchemaFile, 'utf-8')
+                schemaData += '\n' + fs.readFileSync(schemaFile, 'utf-8')
+
+                // Write the result to build/mutations/schema.graphql
+                return path.relative(
+                  this.options.outputDir,
+                  this._writeSubgraphFile(
+                    'mutations/schema.graphql',
```

Comment:
Not going to work on Windows, we need to do `path.join('mutations', 'schema.graphql')` here.